lib/daru.rb

```diff
@@ -58,11 +61,21 @@ def create_has_library(library)
         class_variable_get(cv)
       end
     end
+
+    def plotting_library= lib
+      case lib
+      when :gruff, :nyaplot
+        @plotting_library = lib
+      else
+        raise ArgumentError, "Unsupported library #{lib}"
+      end
+    end
```

Comment:
OK, I will not argue about it more, noting, however, that for me concern separation is broken here. 
Though, it is a part of a larger (and harder) discussion: whether Daru should be looked at as a "new data structure" at a first place + infrastructure (of plotting and analytics and reading/writing) around it; or should it be seen as a main entry point for all data processing functions.

The former seems more future-proof for me, but current development tends to the latter, as far as I can understand.